core/src/mindustry/entities/abilities/MoveLightningAbility.java

```diff
@@ -19,28 +22,69 @@
     public float minSpeed = 0.8f, maxSpeed = 1.2f;
     //Lightning color
     public Color color = Color.valueOf("a9d8ff");
+    //Shifts where the lightning spawns along the Y axis
+    public float offset = 0f;
+    //Jittering heat sprite like the shield on v5 Javelin
+    public TextureRegion heatRegion = Core.atlas.find("error");
```

Comment:
Just make this a `String`. Loading stuff in the constructor will lead to a headless crash.

Reply:
Yea I experienced this while working on Project Unity